src/coreclr/vm/nativeeventsource.cpp

```diff
@@ -189,4 +189,24 @@ extern "C" void QCALLTYPE LogContentionStop(uint8_t ContentionFlags, uint16_t Cl
     END_QCALL;
 }
 
+extern "C" void QCALLTYPE LogWaitHandleWaitStart(uint8_t WaitSource, void* AssociatedObjectID, uint16_t ClrInstanceID)
+{
+    QCALL_CONTRACT;
+    BEGIN_QCALL;
+
+    FireEtwWaitHandleWaitStart(WaitSource, AssociatedObjectID, ClrInstanceID);
+
+    END_QCALL;
+}
+
+extern "C" void QCALLTYPE LogWaitHandleWaitStop(uint16_t ClrInstanceID)
+{
+    QCALL_CONTRACT;
+    BEGIN_QCALL;
+
+    FireEtwWaitHandleWaitStop(ClrInstanceID);
+
+    END_QCALL;
+}
```

Comment:
These are unnecessary since the event is not raised from the managed side in CoreCLR currently